openssl/src/aes.rs

```diff
@@ -258,6 +258,36 @@ pub fn unwrap_key(
     }
 }
 
+/// Encrypts an aes-128/256 block from `in` to `out`.
+/// * `key`: The key-encrypting-key to decrypt the wrapped key. Must be a decrypting key
```

Comment:
key-encrypting-key? wrapped key? decrypting key?

Reply:
These docs should more generally mirror the format used in the rest of the crate.

Reply:
sorry, copy pasted docs, fixed now.